rl4co/models/zoo/ptrnet/decoder.py

```diff
@@ -157,7 +157,8 @@ def forward(
             )
             # select the next inputs for the decoder [batch_size x hidden_dim]
             idxs = (
-                decode_probs(probs, mask, decode_type=decode_type)
+                # note: mask here is the inverse of usual action mask
+                decode_logprobs(log_p, ~mask, decode_type=decode_type)
```

Comment:
maybe change the masking here to be consistent with the rework, i.e.:
- start with a attn_mask of ones (i.e. use torch.ones) here:
https://github.com/ai4co/rl4co/blob/8e9595bf636f5ba5106811a57d58eadd9e255132/rl4co/models/zoo/ptrnet/decoder.py#L147
- gradually set selected actions to False here: https://github.com/ai4co/rl4co/blob/8e9595bf636f5ba5106811a57d58eadd9e255132/rl4co/models/zoo/ptrnet/decoder.py#L75

Reply:
Done!